### PR TITLE
Financial Term Enum

### PR DESCRIPTION
**Pull request by Windchild292**
This moves Financial Term to an enum.

### CODE REVIEW

File: MekHQ/src/mekhq/campaign/finances/enums/FinancialTerm.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    public boolean isQuarterly() {
+        return this == QUARTERLY;
+    }
+
+    public boolean isAnnually() {
+        return this == ANNUALLY;
+    }
+    //endregion Boolean Comparison Methods
+
+    public LocalDate nextValidDate(final LocalDate origin) {
```
Comment (NickAragua) [inline MekHQ/src/mekhq/campaign/finances/enums/FinancialTerm.java:74]: Valid for what?

**Review by NickAragua — APPROVED** [review]
One area that needs some documentation, good otherwise.

File: MekHQ/src/mekhq/campaign/finances/enums/FinancialTerm.java (hunk 1 above)
Comment (Windchild292) [reply MekHQ/src/mekhq/campaign/finances/enums/FinancialTerm.java:74]: Valid for the next financial term... so either a payment is due or a payment made to the company at that date

**Comment by Windchild292** [comment]
Will update for merge post-0.49.3.

**Pull request merged**